src/components/BlogArticle/index.js

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect, useRef} from 'react'
+import {Link} from 'react-router-dom'
+
+import BlogArticleStyled from './BlogArticle'
+import blog from '../../data/blog'
+
+const BlogArticle = ({
+    slug,
+    title,
+    date,
+    author,
+    content
+}) => {
+    const blogContent = useRef(null)
+
+    useEffect(() => {
+        document.title = `${title} - Blog - Instant Visio`
+        blogContent.current.innerHTML = content
+    }, [])
+
+    return(
+        <BlogArticleStyled>
+            <Link to={`/blog/${slug}`} className="article-title"><h2 className="default-title">{title}</h2></Link>
+            <p className="article-dateAuthor">Le {date}, par {author}</p>
```

Comment:
not i18n compatible, e could use Luxon here so support date internationalization through the browser

Reply:
ok, used Luxon